packages/strapi-connector-bookshelf/lib/mount-models.js

```diff
@@ -599,22 +599,31 @@ module.exports = async ({ models, target }, ctx, { selfFinalize = false } = {})
         // Load bookshelf plugin arguments from model options
         this.constructor.__super__.initialize.apply(this, arguments);
 
-        this.on('fetching fetching:collection', (instance, attrs, options) => {
-          populateFetch(definition, options);
-        });
-
-        this.on('saving', (instance, attrs) => {
-          instance.attributes = _.assign(instance.attributes, mapper(attrs));
-        });
-
         const formatValue = createFormatter(definition.client);
         function formatEntry(entry) {
           Object.keys(entry.attributes).forEach(key => {
+            if (key.startsWith('_strapi_tmp_')) {
+              delete entry.attributes[key];
+              return;
+            }
             const attr = definition.attributes[key] || {};
             entry.attributes[key] = formatValue(attr, entry.attributes[key]);
           });
         }
 
+        this.on('fetching fetching:collection', (instance, attrs, options) => {
+          if (Array.isArray(instance.models)) {
```

Comment:
Why are you calling formatEntry before the fetch ?

I think the formatEntry should be only called on the `fetched fetched:collection`

Reply:
Yes, it's an error